tests/user_api/test_export.py

```diff
@@ -27,7 +27,7 @@ def test_user_organization_titles(self):
                 tested = True
                 for ot in ots:
                     self.assertTrue(ot.organization.code)
-                    self.assertTrue(ot.title.code)
+                    self.assertTrue(ot.title is None or ot.title.code)
```

Comment:
It has the possibility that ignores the bug (the case "ot.title is None because of some bugs").
But it'll be difficult to provide the strict test because user setting depends on kintone environment.
So I want to add the notification about it like following.

```py
if ot.title.code is None:
    warnings.warn("ot.title is None. It may user {} doesn't have the title.".format(u.code))
else:
    self.assertTrue(ot.title.code)
```

If you have a good idea, is welcome!

Reply:
> it'll be difficult to provide the strict test because user setting depends on kintone environment

That's right.

> So I want to add the notification about it like following.

Ok. I agree with you.

But the best solution is that this test case provides necessary user settings to kintone, I think.
This is future work.